vendor/vendor.json

```diff
@@ -103,6 +103,12 @@
 			"revision": "8e610b2b55bfd1bfa9436ab110d311f5e8a74dcb",
 			"revisionTime": "2018-06-25T18:44:42Z"
 		},
+		{
+			"checksumSHA1": "B1sVd5XOmjuSFYivGhBd+vPlc1w=",
+			"path": "github.com/ebfe/scard",
+			"revision": "0147d7ead790ac8a5ecc91ee6de68beb7b17c4e9",
+			"revisionTime": "2017-12-31T19:32:11Z"
```

Comment:
There have been ~13 commits to this repo since that revision. Might be better to use the latest

Reply:
it shouldn't be here at all, wtf!?